pkg/model/resource/options.go

```diff
@@ -91,19 +90,13 @@ func (opts *Options) Validate() error {
 	// We can safely look for a '-' as the first char as none of the fields accepts it
 	// NOTE: We must do this for all the required flags first or we may output the wrong
 	// error as flags may seem to be missing because Cobra assigned them to another flag.
-	if strings.HasPrefix(opts.Group, "-") {
-		return fmt.Errorf(groupRequired)
-	}
```

Comment:
We need to keep these checks for v2.

Reply:
Added check for not allowing empty group in the api create validation for v2.

Reply:
See that the above check will not be done. 
IMO we need to create a ValidateV2 which will be the original code and have these changes in the Validate. 
The plugin v2 will call the ValiddateV2 and any upper version will use the Validate. Is it make sense?

Reply:
Done.

Reply:
You can close this one it is solved 👍